src/datasources/prices-api/coingecko-api.service.ts

```diff
@@ -86,7 +86,7 @@ export class CoingeckoApi implements IPricesApi {
         networkRequest: {
           params: {
             vs_currencies: args.fiatCode,
-            contract_address: args.tokenAddress,
+            contract_addresses: args.tokenAddress,
```

Comment:
The title of the PR is not correct in this case, right? The name of the query param was incorrectly set (not the header).

Reply:
Oh, yes, right, I'm changing it, thanks!